e2e/cypress/test_module/cypress/features/cart-page/cart-page.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Cart

```

Comment:
Let's use one approach for naming

![image](https://github.com/user-attachments/assets/199204c8-e035-41c5-9964-f95ba146dfc1)

Reply:
> Let's use one approach for naming
> 
> ![image](https://private-user-images.githubusercontent.com/127191734/351834000-199204c8-e035-41c5-9964-f95ba146dfc1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4NDkwMTQsIm5iZiI6MTcyMTg0ODcxNCwicGF0aCI6Ii8xMjcxOTE3MzQvMzUxODM0MDAwLTE5OTIwNGM4LWUwMzUtNDFjNS05OTY0LWY5NWJhMTQ2ZGZjMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyNFQxOTE4MzRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNWEzNDVlMGM1MTAyNjQ2ZDhiNDJiZjc1Njc3MjdhY2ZmYTA2MjA5M2FiMDNiYzExMTZmZjAwNWYxYzViYjU0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ybkPV6Ni3I1xyJWVk50Og19PExC25PhGDW41zZ2mDLo)

Fixed